### ISSUE TITLE
[java] MoreThanOneLogger is reported on mocked Logger field

### ISSUE THREAD

**Issue by magwas** [pmd/pmd#3860]
**Affects PMD Version:** 
6.42.0

**Rule:**
MoreThanOneLogger
https://pmd.github.io/pmd-6.42.0/pmd_rules_java_errorprone.html#morethanonelogger

**Description:**

Rule 	Violation 	Line
[MoreThanOneLogger](https://pmd.github.io/pmd-6.42.0/pmd_rules_java_errorprone.html#morethanonelogger) 	Class contains more than one logger. 6–10

**Code Sample demonstrating the issue:**

```
package com.kodekonveyor.authentication;

import org.mockito.Mockito;
import org.slf4j.Logger;

class PMDErrorDemonstrationTestBase {

  Logger loggerService = Mockito.mock(Logger.class);

}

```

**Expected outcome:**

PMD reports a violation at line 6-10, but that's wrong. That's a false positive.

**Running PMD through:** Maven

**Comment by oowekyala** [comment]
Thanks for the report. The rule is wrongly written, it flags this field declaration because there are two `Logger` type nodes even though there is a single field.

**Comment by jsotuyod** [comment]
This is no longer reproducible under PMD 7.0.0
